### PR TITLE
Husky

### PR DESCRIPTION
**Pull request by carykees98**

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -6,43 +6,48 @@
...
+  - [talks]()
 
 - [Mirror](./mirror/introduction.md)
-	- [Hardware]()
-	- [Software]()
+  - [Hardware]()
+  - [Software]()
 
-# Guides 
+# Equipment
```
Comment (Alextopher) [inline src/SUMMARY.md:45]: In my opinion Equipment should be before Services.

**Comment by Alextopher** [comment]
I'm sure something like VS code is messing with the indentation. Do you think the indentation you're suggesting in this PR is good and we should require it moving forward? Is there some way you can tell vscode to enforce it?

**Comment by carykees98** [comment]
> I'm sure something like VS code is messing with the indentation. Do you think the indentation you're suggesting in this PR is good and we should require it moving forward? Is there some way you can tell vscode to enforce it?

VS code is indeed responsible for indentation changes but I do think it is a good idea. It improves overall readability and also just overall makes it more clear what is a subpage and what is a category

**Comment by carykees98** [comment]
I can go through and update the indentation for the rest of the book after I merge this

**Comment by carykees98** [comment]
So long as you also think it is a good idea

**Comment by Alextopher** [comment]
I like the idea of standardizing the indentation. There might be opportunity to standardize other things. Such as how we write `updated: xxxx` on every page.

**Pull request closed without merging**